app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -107,6 +108,7 @@ fun MessageItem(
     onFailedMessageRetryClicked: (String) -> Unit = {},
     onFailedMessageCancelClicked: (String) -> Unit = {},
     onLinkClick: (String) -> Unit = {},
+    isSearchActive: Boolean = false,
```

Comment:
suggestion: maybe give it a more universal name? like `isContentClickable` or something like that? 😄